app/Http/Controllers/Admin/Activity/ActivityController.php

```diff
@@ -136,10 +151,17 @@ public function show($id): View|JsonResponse|RedirectResponse
             $transactions = $this->transactionService->getActivityTransactions($activity->id);
             $status = $activity->element_status;
             $progress = $this->activityService->activityPublishingProgress($activity);
+            $coreCompleted = isCoreElementCompleted(array_merge(['reporting_org' => $activity->organization->reporting_org_complete_status], $activity->element_status));
+            $iatiValidatorResponse = null;
```

Comment:
use ternary

$validatorResponse = $this->activityValidatorResponseService->getValidatorResponse($id);
$iatiValidatorResponse = ($validatorResponse)?$validatorResponse->response:null;